components/admin_console/system_user_detail/system_user_detail.tsx

```diff
@@ -451,21 +475,18 @@ export default class SystemUserDetail extends React.PureComponent {
                 <div className='admin-console-save'>
                     <SaveButton
                         saving={this.state.saving}
-                        disabled={!this.state.saveNeeded || (this.canSave && !this.canSave())}
+                        disabled={!this.state.saveNeeded}
                         onClick={this.handleSubmit}
                         savingMessage={Utils.localizeMessage('admin.saving', 'Saving Config...')}
                     />
                     <div
                         className='error-message'
                         ref='errorMessage'
-                        onMouseOver={this.openTooltip}
-                        onMouseOut={this.closeTooltip}
                     >
                         <FormError error={this.state.serverError}/>
                     </div>
                     <Overlay
                         show={this.state.errorTooltip}
-                        delayShow={Constants.OVERLAY_TIME_DELAY}
```

Comment:
Why this is removed?

Reply:
Overlay doesn't have a property `delayShow` so type-check marks this as wrong